### PR TITLE
global: module skeleton

### LINKED ISSUE #1: global: initial package structure
Introduce initial package structure similarly to CWL or Yadage workflow engines.

### PR DESCRIPTION
**Pull request by dinosk**
(Closes #1)

Signed-off-by: Dinos Kousidis <dinos.kousidis@cern.ch>

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Changes
+=======
+
+Version 0.2.0 (2018-04-17)
```
Comment (tiborsimko) [inline CHANGES.rst]: Please leave version and date on `FIXME` as we shall be releasing for later versions...

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    setup_requires=setup_requires,
+    tests_require=tests_require,
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```
Comment (tiborsimko) [inline setup.py]: We should aim at 3.using Python 3.6... 2.7 is only there "historically" for Yadage etc.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+sudo: true
+
+language: python
+
+cache:
+  - pip
+
+python:
+  - "2.7"
```
Comment (tiborsimko) [inline .travis.yml]: Better use 3.6 from the start.

File: reana_workflow_engine_serial/__init__.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""REANA-workflow-engine-serial."""
```
Comment (tiborsimko) [inline reana_workflow_engine_serial/__init__.py]: In free text context, the "official" component names are capitalised, i.e. `REANA-Workflow-Engine-Serial`.

File: reana_workflow_engine_serial/models.py [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""Models for REANA Workflow Engine Serial."""
```
Comment (dinosk) [inline reana_workflow_engine_serial/models.py]: The models should be imported from reana-db, but until then I added this for development purposes

Comment (tiborsimko) [reply reana_workflow_engine_serial/models.py]: See the incorrect import order failure on Travis CI... Can you please correct? Or simply comment out the `isort` directive in `run-tests.sh` for now?

**Comment by diegodelemos** [comment]
- [ ] Update job status using reana-workflow-controller REST API.

**Comment by diegodelemos** [comment]
Superseded by https://github.com/reanahub/reana-workflow-engine-serial/pull/4.

**Pull request closed without merging**